src/app.rs

```diff
@@ -59,14 +60,33 @@ impl Application for AppModel {
         let mut tasks = vec![];
 
         let about = About::default()
-            .set_application_name(fl!("app-title"))
-            .set_application_icon(Self::APP_ID)
-            .set_developer_name("COSMIC Utilities")
-            .set_version(env!("CARGO_PKG_VERSION"))
-            .set_license_type("GPL-3.0")
-            .set_repository_url(REPOSITORY)
-            .set_support_url(format!("{REPOSITORY}/issues"))
-            .set_developers([("Dexter Reed".into(), "dreed4470@proton.me".into())]);
+            .name(fl!("app-title"))
+            .icon(Self::APP_ID)
+            .author("COSMIC Utilities")
```

Comment:
You should be the author, we could add an organization method to link a project to an organization